doc/dependency-resolution.md

```diff
@@ -1,356 +1,129 @@
-## Dependency Resolution
+# Dependency Resolution
 
-**Problem Statement**
+Dependency Resolution is a concept within the Publishing API to describe two
+processes:
 
-We're undergoing a migration to the new publishing platform. The publishing
-platform doesn't currently prescribe how the following features should work:
+- Determining the links that are presented with a content item when it is
+  represented in the Content Store.
```

Comment:
I think of this as link expansion, which is a related, but separate concept to Dependency Resolution.